content/docs/examples/advanced-egress/egress-gateway-mtls-origination/index.md

```diff
@@ -0,0 +1,533 @@
+---
+title: Mutual TLS origination by Egress Gateway
+description: Describes how to configure an Egress Gateway to perform mutual TLS origination to external services.
+weight: 45
+keywords: [traffic-management,egress]
+---
+
+The [Configure an Egress Gateway](/docs/examples/advanced-egress/egress-gateway) example describes how to configure
+Istio to direct the egress traffic through a dedicated service called _Egress Gateway_. This examples shows how to
+configure an Egress Gateway to perform mutual TLS origination to external services.
+You will deploy an [NGINX](https://www.nginx.com/) server in your Kubernetes cluster without injecting Istio sidecar
+proxy into it.
+This server will simulate a server outside the Istio service mesh. Then you will configure the Egress Gateway to
+perform mutual TLS with this server.
+Finally, you will direct the traffic from the application pods inside the mesh to the server outside the mesh through
+the Egress Gateway. The Egress Gateway will perform mutual TLS origination with the NGINX server.
+
+## Before you begin
+
+This examples assumes you deployed Istio with [mutual TLS Authentication](/docs/tasks/security/mutual-tls/)
+enabled. Follow the steps in the [Before you begin](/docs/examples/advanced-egress/egress-gateway/#before-you-begin)
```

Comment:
I dont think we need this sentence anymore. mututal tls is incrementally enabled. i.e. its always enabled in some way,

Reply:
When configuring a Gateway, the Gateway should know to expect mTLS or not. Otherwise, double instructions must be provided: "if you have mTLS enabled, create this definition, otherwise create that definition".

```
tls:
           mode: MUTUAL
           serverCertificate: /etc/certs/cert-chain.pem
           privateKey: /etc/certs/key.pem
           caCertificates: /etc/certs/root-cert.pem
```